components/script/dom/eventsource.rs

```diff
@@ -581,7 +594,7 @@ impl EventSourceMethods<crate::DomTypeHolder> for EventSource {
         );
         // Step 11
         request.cache_mode = CacheMode::NoStore;
-        // Step 12
+        // Step 13
```

Comment:
Thanks for updating the numbers, but do you mind also includes specification text here and below?

Reply:
Done.